url_builder_integration.py

```diff
@@ -0,0 +1,443 @@
+# -*- coding: utf-8 -*-
+"""URL_integration.ipynb
+
+Automatically generated by Colaboratory.
+
+Original file is located at
+    https://colab.research.google.com/drive/1J17Kp-vGlOxUBAPkvMVKZd9rTl7rGVMq
+"""
+
+import time
+import pandas_gbq
+import pandas as pd
+from datetime import datetime
+from google.cloud import bigquery
+from google.oauth2 import service_account
+
+from urlbuilder import URLBuilder as ScieloURLBuilder
+from urlbuilder import ScieloSearchLocations
+
+import centaurminer as mining
+import os
+
+class URL_builder():
+  """
+    URL_builder class uses `urlbuilderfactory` method
+    to extract URLs from a specific journal.
+
+    ...
+
+    Attributes
+    ----------
+    search_word (str): word to be searched
+
+    Methods
+    -------
+    url_collector(): Scrapes URLs from the pages.
+    create_url_schema(list_of_urls,catalog_url,is_pdf,lang): Create url schema to push into bigquery.
+    send_to_bigquery(project_id,table_id): To push url_schema to bigquery.
+    
+  """
+  
+  def __init__(self,search_word):
+    """
+        Constructs all the necessary attributes for the URL_builder object.
+
+        Parameters
+        ----------
+            search_word(str): word to be searched
+        """
+    self._url_list=[]
+    self.total_urls=[]
+    self.valid_urls=[]
+    self._search_word=search_word
+    self._driver_path = '/usr/lib/chromium-browser/chromedriver'
+    self._anchor_element={'arxiv':'span.list-identifier > a','biorxiv':'a.highwire-cite-linked-title','medrxiv':'a.highwire-cite-linked-title','preprint':'a#title.title',
+            'pbmc':'a.class1','jamanetwork':'h3.article--title > a'}
```

Comment:
do we need to add `scielo` here too?

Reply:
Nope, as we are copying Scielo URL_builder from Adrian.